src/app/qgsmeasuredialog.cpp

```diff
@@ -718,6 +718,7 @@ void QgsMeasureDialog::repopulateComboBoxUnits( bool isArea )
     mUnitsCombo->addItem( QgsUnitTypes::toString( Qgis::DistanceUnit::Millimeters ), static_cast< int >( Qgis::DistanceUnit::Millimeters ) );
     mUnitsCombo->addItem( QgsUnitTypes::toString( Qgis::DistanceUnit::Inches ), static_cast< int >( Qgis::DistanceUnit::Inches ) );
     mUnitsCombo->addItem( QgsUnitTypes::toString( Qgis::DistanceUnit::Degrees ), static_cast< int >( Qgis::DistanceUnit::Degrees ) );
+    mUnitsCombo->addItem( QgsUnitTypes::toString( Qgis::DistanceUnit::ChainsInternational ), static_cast< int >( Qgis::DistanceUnit::ChainsInternational ) );
```

Comment:
any particular reason to add that particular one among the gazillions new ones ;-) ?
By the way the above lines could benefit from being rewritten as following, to make code smaller and avoid potentially copy&paste errors:
```
for( Qgis::DistanceUnit unit : { Qgis::DistanceUnit::Meters, .... })
{
    mUnitsCombo->addItem( QgsUnitTypes::toString( unit ), static_cast< int >( unit ) );
}
```

Reply:
There was recent discussion on mastodon with real world use cases for chains. I think exposing others is just clutter until there's actually someone asking for them 😂

Reply:
Got it. My point was not to add all of them, but "why do we need to "en-chain" mortals ?" :-)